repo/mongodb/repo.go

```diff
@@ -207,6 +207,22 @@ func (r *Repo) Remove(ctx context.Context, id eh.UUID) error {
 	return nil
 }
 
+// Update lets the function update data in the collection.
+func (r *Repo) Update(ctx context.Context, f func(*mgo.Collection) error) error {
```

Comment:
As it's not doing anything extra like in `FindCustom` and we might do something else than Update, maybe it should be something like `Collection()` or `Session()`?